modules/nodes-scheduler-descheduler-create-role.adoc

```diff
@@ -38,27 +38,27 @@ rules:
 . Create the service account that runs the job:
 +
 ----
-# oc create sa <file-name>.yaml -n kube-system
+# oc create sa <file-name>.yaml -n openshift-operators
```

Comment:
IIRC, it should be in descheduler-operator project

Reply:
@ravisantoshgudimetla I am seeing that the Descheduler Operator and CRD are defaulting to the `openshift-operators` project. I just did an install.